pkg/apis/k0s/v1beta1/cplb_linux.go

```diff
@@ -30,7 +31,7 @@ func getDefaultNIC() (string, error) {
 	}
 
 	for _, route := range routes {
-		if route.Dst == nil {
+		if route.Dst.IP.Equal(net.IPv4zero) || route.Dst.IP.Equal(net.IPv6zero) {
```

Comment:
Is there no chance that anything in the "path" on `route.Dst.IP` could be `nil`?

Reply:
I don't think that's possible, but it doesn't hurt so I added it back.